src/scicookie/{{cookiecutter.project_slug}}/build-system/skcdemo.cpp

```diff
@@ -4,6 +4,6 @@ namespace py = pybind11;
 
 PYBIND11_MODULE(skcdemo, m) {
     m.def("hello", [](){
-        py::print("Hello, scikit-build-core!");
+        py::print("Hello !!!");
```

Comment:
maybe you should change the name of the file to something more generic than `skcdemo`
maybe example.cpp ... or as it uses pybind11 .. maybe pybind11.cpp ..